client/src/utils/robotMetadataUtils/robotMetadataUtils.test.js

```diff
@@ -4,13 +4,15 @@
 
 jest.mock('../../api/ssot');
 jest.mock('../downloadString');
-jest.mock('../attributeAndParamUtils');
+jest.mock('../localSsot/parameters');
```

Comment:
Are all these needed? From the changes it seems that you are only mocking the localSsot

Reply:
You are right. We only need jest.mock('../localSsot/ssot'). I deleted the others ✅